src/tests/ftest/io/ior_hdf5_small.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2018-2019 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+The Government's rights to use, modify, reproduce, release, perform, display,
+or disclose this software are subject to the terms of the Apache License as
+provided in Contract No. B609815.
+Any reproduction of computer software, computer software documentation, or
+portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from ior_test_base import IorTestBase
+
+
+class IorHdf5Small(IorTestBase):
```

Comment:
Any specific reason this was not added as part of ior_small.py? As that test takes care of all the other ior api's it will be nice to add HDF5 to the same test as well.

Reply:
only reason was to not affect ior small test in any way.